packages/mdc-textfield/_mixins.scss

```diff
@@ -126,6 +164,10 @@
   @include mdc-text-field-hover-outline-color($mdc-text-field-error-on-light);
   @include mdc-text-field-focused-outline-color($mdc-text-field-error-on-light);
 
+  // Textarea specific mixins
```

Comment:
move this to `mdc-text-field-textarea-invalid_` mixin. Its at HEAD of master now

Reply:
Should we move all of the variants to their own invalid mixins? Textfield is the only one that has an -invalid mixin.

I opted to condense everything here, but if we break out textarea, shouldn't we also break out outline ? 
textfield mixin in master
https://github.com/material-components/material-components-web/blob/fc3a9d5edce204eeb9c5a74a914d39e147ed300f/packages/mdc-textfield/_mixins.scss#L341

invalid mixin in master 
https://github.com/material-components/material-components-web/blob/fc3a9d5edce204eeb9c5a74a914d39e147ed300f/packages/mdc-textfield/_mixins.scss#L125